Document how to resolve `Cannot find module 'react/jsx-runtime'` error in VS Code

If you come across the following error;
```
Cannot find module 'react/jsx-runtime' or its corresponding type declarations.ts(2307)
```
then you need to select the proper typescript version to use the workspace version:
1. Press `ctrl+shift+p` in a TypeScript file
2. Choose "Select TypeScript Version"
3. Pick "Use Workspace Version"

--- a/documentation/docs/contribute/development-guide/01-setup.md
+++ b/documentation/docs/contribute/development-guide/01-setup.md
@@ -143,4 +143,6 @@ To let the IDE know that dependencies can be found in `.yarn` instead of `node_m
 
 #### Yarn PnP in VSCode
 
-You'll might need to add `"typescript.tsdk": "./template-fastapi-react/web/.yarn/sdks/typescript/lib"` to your workspace settings to make typescript work. Other than that, the committed .vscode folder should contain all the configurations you'll need.
+Because the project uses Yarn PnP, VSCode's language server must use the same typescript executable as the project in order to access dependencies. This is located in `web/.yarn/sdks/typescript/lib`. To use this version, simply open a `.ts(x)` file, open the command palette (default `Ctrl+Shift+P` or `Cmd+Shift+P`), find `TypeScript: Select TypeScript Version...` and select `Use Workspace Version`. 
+
+Not using the correct typescript executable might cause `Cannot find module '***' or its corresponding type declarations.` and `JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists.` errors.

